app/models/route.rb

```diff
@@ -131,8 +131,8 @@ def validate_external_target(target)
 
     return unless errors[:redirect_to].empty? # Don't continue, as the host validation may fail
 
-    errors[:redirect_to] << "external domain must be within .gov.uk or british-business-bank.co.uk" unless
-      uri.host.end_with?(".gov.uk") || uri.host == "british-business-bank.co.uk"
+    errors[:redirect_to] << "external domain must be within .gov.uk, .judiciary.uk or british-business-bank.co.uk" unless
+      uri.host.end_with?(".gov.uk") || uri.host.end_with?(".judiciary.uk") || uri.host == "british-business-bank.co.uk"
```

Comment:
Minor: you can call `end_with?` with [multiple arguments](https://apidock.com/ruby/String/end_with%3F) which would made this conditional a bit easier to read.

Reply:
Oh oops, I merged this in before your comment! It's a good point though, I wasn't aware of that.